.nvmrc

```diff
@@ -0,0 +1 @@
+4
```

Comment:
intended?

Reply:
Yup, I haven't tested whether it works with node 8 but I know it works with node 4. We can update at a later date.